### PR TITLE
chore(prettier): add prettier and prettierrc to template

### PR DESCRIPTION
**Pull request by TheAmazingPT**
All prettier options: https://prettier.io/docs/en/options.html

### CODE REVIEW

**Comment by TheAmazingPT** [comment]
Shall I add a run script in the `template/package.json`?
The idea is now kinda that the editor can detect the `.prettierrc` and formats the current file on save. What do you think? There were some complaints on having this task in the build steps.

**Comment by Scarysize** [comment]
I wouldn't add a run script. To migrate a project to prettier its often enough to do a `find . | prettier` run once and then rely on the editor picking up the `.prettierrc` file.

**Comment by aemkei** [comment]
Is there a VSC extension that takes the prettierrc settings into account and let you do that using a command?

**Comment by pwambach** [comment]
yep https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode uses the config file.

I also activated the `prettier.requireConfig` setting, so that projects without a config file will not be formatted.

**Comment by pwambach** [comment]
One more thing, when prettier handles the 80 chars line length, can we switch off the eslint setting for this? Prettier sometimes cannot split the lines and then eslint complains. We should let prettier do it as good as it can...

File: templates/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "printWidth": 80,
```
Comment (pwambach) [inline templates/.prettierrc]: disable eslint line width rule please

**Pull request merged**